docs/changelog/106032.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 106032
+summary: Unattended transforms can still go into the failed state
```

Comment:
This PR is not the full solution right? It just adds a new variable to the task params and a followup will actually fix the bug?

If that's correct then this PR should be `>non-issue` instead of `>bug`, and not have a release note. Changing the labels should remove this file. Then the PR that actually has the fix can have the release note. We don't want the user-facing release notes to be bloated by multiple items that are all part of the same fix.

Reply:
That is correct - I changed the labels, and it looks like the changelog was deleted.  Thank you (that also helps me understand why that file sometimes gets added and other times does not)